Source/OxyPlot/Series/AreaSeries.cs

```diff
@@ -75,7 +75,7 @@ public AreaSeries()
         /// Gets the actual color of the line for the second data set.
         /// </summary>
         /// <value>The actual color.</value>
-        public OxyColor ActualColor2
+        public virtual OxyColor ActualColor2
```

Comment:
I find it a bit strange that only the ActualColor2 property is virtual (why isn't ActualColor and ActualFill virtual...) But I see it is useful for the TwoColorAreaSeries.